lib/gallactickeys.js

```diff
@@ -117,6 +117,25 @@ gkeys.inspectAccount = (privateKey) => {
   return account;
 }
 
+/**
+ * Get account information by publickey
+ * @param {String} publicKey of the account
+ * @returns {Object} [contains privateKey, publicKey, accountAddress and validatorAddress]
```

Comment:
it doesn't contain private key. ;)

Reply:
yeah you are right. Corrected in commitId `3d8f6f6`